src/nft/components/collection/UnavailableCollectionPage.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from 'test-utils/render'
+
+import { UnavailableCollectionPage } from './UnavailableCollectionPage'
+
+describe('Nonexistent Collection', () => {
```

Comment:
have you considered a snapshot test here?

Reply:
Considered a snapshot test, but I think manually querying the component and verifying specific requirements tends to be more robust and works for this case.